lib/codebreaker_gem/game.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+require_relative 'validation/validation'
+require_relative 'validation/errors_include'
+
+module CodebreakerGem
+  class Game
+    include Validation
+    attr_accessor :name, :code, :difficulty, :attempts, :hints, :available_hints
+
+    CODE_RANGE = (1..6).to_a.freeze
+    CODE_LENGTH = 4
+    USER_NAME = ''
+    GAME_CODE = ''
+    GAME_DIFFICULTY = 0
+    GAME_ATTEMPTS = 0
+    GAME_HINTS = 0
+    DIFFICULTIES = {
+      easy: { attempts: 15, hints: 2 },
+      medium: { attempts: 10, hints: 1 },
+      hell: { attempts: 5, hints: 1 }
+    }.freeze
+
+    def initialize
+      @name = USER_NAME
+      @code = GAME_CODE
+      @difficulty = GAME_DIFFICULTY
+      @attempts = GAME_ATTEMPTS
+      @hints = GAME_HINTS
```

Comment:
def initialize(name: USER_NAME, code: GAME_CODE, ...)
u can use initialize in specs and u won`t need accessors